src/cdk/coercion/css-property.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+/** Coerces a value to a string that can be assigned as a CSS property. */
+export function coerceCssProperty(value: any): string {
```

Comment:
I think we either need to make it work with different units and call the function `coerceCssUnitValue` or it can stay just pixels and be `coerceCssPixelValue`

Reply:
I'd rather keep it to pixels so we don't have to keep up with the different units. Changed the naming and we can reconsider if somebody requests it.

Also TIL that CSS frequency units: https://developer.mozilla.org/en-US/docs/Web/CSS/frequency